docs/todo.rst

```diff
@@ -1,8 +1,47 @@
-.. _todo-label:
+..  _todo-label:
 
-Open issues
+Contributing
 ---------------
 
+If you want to develop for Asterix, you might need packages to test and build the documentation. Run:
+
+.. code-block:: bash
+    
+    $  pip install -e '.[dev,docs]'
+
+To contribute to Asterix, please follow the following steps:
+    1. Make sure your local `master` branch is up-to-date by pulling.
+    2. Create a new branch off `master` with a name of your choice and commit your work.
+    3. When done, open a PR and request a review after ensuring your branch is up-to-date with the base branch you're merging into (usually `master`) and after running the pytests and the linter locally. You can then the run the pytests
```

Comment:
"You can then the run the pytests" has a "the" too many and I would put a colon after "pytests", like you did after the flake8 linter further below.

Actually, even so it's a little confusing with the previous sentence ("[...] after running the pytest and the linter locally." So maybe Just do:
"[...] after running the pytests and the linter locally as follows. To run the pytests:
```bash
$ pytest
```
To run the flake8 linter:
```bash
$ flake8 . --max-line-length=121 --count --statistics
```
"
It might also be useful to keep the instructions where in the repo these need to be run - I know I like to come back to that because I need to refresh my memory sometimes.